modules/plugins/languages/yaml.nix

```diff
@@ -0,0 +1,67 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib;
+with builtins; let
```

Comment:
Please never do this. Not only do we disallow `with *` patterns in nvf, it is generally really bad practice.

Also `lib` already includes most of the builtins as alieases, so you're introducing overhead without a benefit.